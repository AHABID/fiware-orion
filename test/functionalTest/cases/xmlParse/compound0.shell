# 0. ok:  No compound value
echo "0: +++++++++++++++++++++++++++++++++"
url='/ngsi10/updateContext'
payload='<?xml version="1.0"?>
<updateContextRequest>
  <contextElementList>
    <contextElement>
      <entityId type="Room" isPattern="false">
        <id>ConferenceRoom</id>
      </entityId>
      <contextAttributeList>
        <contextAttribute>
          <name>temperature</name>
          <type>degree</type>
          <contextValue>1</contextValue>
        </contextAttribute>
      </contextAttributeList>
    </contextElement>
  </contextElementList>
  <updateAction>APPEND</updateAction>
</updateContextRequest>'
curlXml ${url} "${payload}"

