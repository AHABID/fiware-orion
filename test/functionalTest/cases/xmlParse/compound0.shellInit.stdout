{ "dropped" : "testharness", "ok" : 1 }
T@17:45:10  paConfigActions[727]: setting trace levels to '0-1'
E@17:45:10  exitFunc[1009]: unlink(/tmp/orion_9999.pid): No such file or directory
F@17:45:10  ***[1063]: Connected to mongo at localhost:testharness
V@17:45:10  restStart[569]: Starting HTTP daemon on IPv4 0.0.0.0 port 9999
V@17:45:10  restStart[611]: Starting HTTP daemon on IPv6 :: port 9999
2@17:45:11  xmlRequestGet[155]: Found xmlRequest of type 17, method 'POST' - index 12 (/updateContextRequest)
F@17:45:11  ***[981]: In sigHandler - caught signal 15
X@17:45:11  sigHandler[987]: Received signal 15
